.github/workflows/codeql.yml

```diff
@@ -0,0 +1,58 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ "v1", "cloud-*", "master", "release/*" ]
+  pull_request:
+    branches: [ "v1", "cloud-*", "master", "release/*" ]
+  schedule:
+    - cron: '36 17 * * 0'
+  workflow_call:
+    inputs:
+      ref:
+        required: true
+        type: string
+
+jobs:
+  analyze:
+    name: Analyze (${{ matrix.language }})
+    runs-on: ${{ (matrix.language == 'swift' && 'macos-latest') || 'ubuntu-latest' }}
+    timeout-minutes: ${{ (matrix.language == 'swift' && 120) || 360 }}
```

Comment:
Do we need the condition for "swift"?

Reply:
We're going to follow up with some cleanup, this is just to get something in place.  :smile: